alipay.php

```diff
@@ -33,53 +23,15 @@ class Alipay extends NonmerchantGateway
      */
     public function __construct()
     {
+        $this->loadConfig(dirname(__FILE__) . DS . 'config.json');
```

Comment:
The link to the documentation on line 6 doesn't appear to work anymore, so might as well update it to wherever their current documentation is